packages/ember-cli-code-coverage/index.js

```diff
@@ -72,7 +88,12 @@ module.exports = {
     }
 
     const IstanbulPlugin = require.resolve('babel-plugin-istanbul');
-    return [[IstanbulPlugin, { cwd, include: '**/*', exclude }]];
+    return [
+      useStringLookupForPlugin
```

Comment:
@vstefanovic97 could you please explain this ternary? can't we do e.g. `require` all the time?

Reply:
There is a problem with [babel plugin serialization](https://github.com/embroider-build/embroider/issues/1525) in v3, I solved it using string lookups since I found [this](https://github.com/Serabe/ember-bind-helper/pull/21) as a reference.
I tried to use string lookup all the time, but it didn't work for v1 and v2

Reply:
@SergeAstapov Added comment about this to the code